src/admin/components/elements/Autosave/index.tsx

```diff
@@ -136,13 +139,28 @@ const Autosave: React.FC<Props> = ({ collection, global, id, publishedDocUpdated
     }
   }, [publishedDocUpdatedAt, versions]);
 
+  // dynamically import date-fns locale for 'last saved' time
+
+  useEffect(() => {
+    const importLocale = async () => {
+      const localeToSet = await import(
```

Comment:
Should skip dynamic import when locale is `en` since en-US is imported by default